### PR TITLE
DEV-12539: Add values for mandatory OrderRequest parameters in Orders tests

### PR DESCRIPTION
**Pull request by lawrenceFinbourne**
… tests

# Pull Request Checklist

- [x] Read the [contributing guidelines](../blob/master/docs/CONTRIBUTING.md)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

**Review by matthew-boardman — APPROVED** [review]

File: sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs [hunk 1]
```diff
@@ -144,7 +144,11 @@ public void Upsert_Simple_Order()
                             { $"Order/{testScope}/Account", new PerpetualProperty($"Order/{testScope}/Account", new PropertyValue("J Wilson")) },
                             { $"Order/{testScope}/Strategy", new PerpetualProperty($"Order/{testScope}/Strategy", new PropertyValue("RiskArb")) },
                         },
-                        side: "Buy"
+                        
+                        side: "Buy", 
+                        state: "New", 
+                        type: "Limit", 
+                        date: DateTimeOffset.Parse("2022-07-02")
```
Comment (rizwansaeed) [inline sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:151]: are/should these be cut labels?

Comment (lawrenceFinbourne) [reply sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:151]: I don't know what a cut label is

Comment (rizwansaeed) [reply sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:151]: @timbarrass-finbourne should these be cut labels?

Comment (lawrenceFinbourne) [reply sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs:151]: Cut labels aren't supported for order entity date fields at present

**Pull request merged**